### PR TITLE
Toggle Player Name on Map

### PR DESCRIPTION
**Pull request by StefArma**
I added the button correctly

but i'm sure i messed up with the code output. 
It works with a single click, but if i click twice the variable looks disappearing.

### CODE REVIEW

File: AntistasiOfficial.Altis/Stringtable.xml [hunk 1]
```diff
@@ -708,6 +708,12 @@
 		</Container>
 
 		<Container name="Admin">
+			<Key ID="STR_UI_ADMIN_TGLPN_TEXT">
+				<Original>Map names</Original>
+			</Key>
+			<Key ID="STR_UI_ADMIN_TGLPN_TT">
+				<Original>Toggle player names displayed on map</Original>
```
Comment (IrLED) [inline AntistasiOfficial.Altis/Stringtable.xml]: What about `Map marker`s instead of `Map names`

Comment (StefArma) [reply AntistasiOfficial.Altis/Stringtable.xml]: Best would be Map player names

File: AntistasiOfficial.Altis/UI/defines/menu_admin.hpp [hunk 2]
```diff
@@ -16,8 +16,10 @@ else {server setVariable ['enableWpnProf',true,true]; (_this select 0) ctrlSetTe
 #define ADMIN_TGLTM "server setVariable ['testMode', [true, false] select (server getVariable ['testMode', false]), true]; \
 	(_this select 0) ctrlSetTextColor ([[1,0,0,1], [0.18,0.545,0.341,1]] select (server getVariable ['testMode', false]))" \
 
+#define ADMIN_TGLPN "if(server getVariable [""hardMode"",false]) then {server setvariable [""hardmode"",flase,true]; (_this select 0) ctrlSetTextColor [1,0,0,1];} else {server setvariable [""hardmode"",true,true];(_this select 0) ctrlSetTextColor [0.18,0.545,0.341,1];};"
+
```
Comment (StefArma) [inline AntistasiOfficial.Altis/UI/defines/menu_admin.hpp]: This is working partially, and missing the colour switch as line 3 of this file

Comment (IrLED) [reply AntistasiOfficial.Altis/UI/defines/menu_admin.hpp]: Initial button coloring looks like happening somewhere here https://github.com/A3Antistasi/antistasiofficial/blob/99447d9a6bb629c24291721c2e9c59b148f2a05b/AntistasiOfficial.Altis/UI/functions/fn_UI_createMenu.sqf#L71-L81

**Pull request merged**